api/v1beta1/infraenv_types.go

```diff
@@ -202,6 +202,9 @@ type BootArtifacts struct {
 	// IpxeScriptURL specifies an HTTP/S URL that contains the iPXE script
 	// +optional
 	IpxeScriptURL string `json:"ipxeScript"`
+	// DiscoveryIgnitionURL specifies and HTTP/S URL that contains the discovery ignition
```

Comment:
Sorry just noticed this tiny bit 
```suggestion
	// DiscoveryIgnitionURL specifies an HTTP/S URL that contains the discovery ignition
```
Looks good otherwise!

Reply:
oops